sdk/cognitivelanguage/ai-language-text/test/public/utils/customTestHelpter.ts

```diff
@@ -108,8 +110,8 @@ async function getAssetsforProject(
       break;
   }
 
-  dir = path.join(pathName, files);
-  return assets;
+  const dirName = path.join(pathName, files);
```

Comment:
Can we do the same for assets and files?

Reply:
Since assets and files are initialized through "switch" statement, I'm not sure if we can pull them out into constants

Reply:
Oh that makes sense. Thank you for the clarification.

Reply:
Now that looks much better and is easily readable, thanks! 👍